### PR TITLE
fix: add more default arguments to PagesMenu::build method

### LINKED ISSUE #2922: [BUG] Timber::get_pages_menu() throws warnings when using Polylang
### Expected Behavior

Expected to not get any warning messages.

### Actual behavior

Warning:
Undefined array key "menu" in
wp-content\mu-plugins\polylang\frontend\frontend-nav-menu.php
on line 280

Warning:
Undefined array key "theme_location" in
wp-content\mu-plugins\polylang\frontend\frontend-nav-menu.php
on line 294

### Steps to reproduce behavior

_No response_

### Notes

_No response_

### What version of Timber are you using?

2.0

### What version of WordPress are you using?

6.4.3

### What version of PHP are you using?

8.1

### How did you install Timber?

Installed or updated Timber through Composer

### PR DESCRIPTION
**Pull request by Levdbas**
Related:

- #2922
- Ticket 2

## Issue
Polylang pages menus assume default arguments are set, but they are not in PagesMenu.php

## Solution
Add those two default arguments.

## Impact
Better compat with Polylang.

## Usage Changes
no.

## Considerations
no.

## Testing
@lnorby and @cafesk8, are you able to test this code to see if it works as expected? You can just copy/paste the code of the changed file to your local codebase.

### CODE REVIEW

**Review by gchtr — DISMISSED** [review]
Seems like a pragmatic fix.

Maybe we could add a comment for these two arguments that explains that we set them because of Polylang?

**Comment by Levdbas** [comment]
> Seems like a pragmatic fix.
> 
> Maybe we could add a comment for these two arguments that explains that we set them because of Polylang?

Agreed, added to docblock!

**Review by gchtr — DISMISSED** [review]

**Comment by gchtr** [comment]
@Levdbas Very good, thanks!

File: src/PagesMenu.php [hunk 1]
```diff
@@ -33,6 +33,8 @@ public static function build($menu, $args = []): ?self
          * Default arguments from wp_page_menu() function.
          *
          * @see wp_page_menu()
+         *
+         * @since 2.3.0 The 'menu' and 'theme_location' are added to provide compatibility with Polylang. See ticket #2922.
```
Comment (nlemoine) [inline src/PagesMenu.php]: Maybe use `@see {fullURL}`, avoid the pain of manually look for the ticket number :)

**Comment by cafesk8** [comment]
@gchtr  - seems to be working as expected - IMHO you can merge this

**Review by gchtr — APPROVED** [review]

**Pull request merged**